events/forms.py

```diff
@@ -16,3 +16,9 @@ class Meta:
 class SearchForm(forms.Form):
     search = forms.CharField(label='',widget=forms.TextInput(attrs={'placeholder': 'search for events'}),max_length=100, required=False)
     show_old = forms.BooleanField(label='show old events', required=False)
+    CHOICES=[   ("1", "Most Popular"),
+                ("2", "Least Popular"),
+                ("3", "Upcoming"),
+                ("4", "Date Created"),
```

Comment:
A naming convention like DATEASC, DATEDEC would be easier than referencing the definition for the number.

Reply:
After testing a bunch, I determined that that first parameter(the numbers) is the value that is sent through the GET request. And I use that value in the views(if statements) and also my widget when I output the form. If I changed it to a naming convention, there wouldn't be a problem in the views file ( since I can just compare the string), however, I use the numerical value in the for loop inside the sortOptions.html (checking it against a forloop counter) in order to add a "active" class attribute to whatever button the user pressed before submitting. 

Now all of this is just styling preference. But basically if I do switch it to a naming convention, I don't think I can style the sort options with a button toggle and it would just be a basic line of radio buttons. 

But try it out and let me know what you think.

Reply:
Hmm. Kinda hacky but I'll allow it. I think it would involve creating 2 widgets: A ButtonGroup widget and a Button widget. The button widget would have an 'active' property that the button group could call on.